libr/core/linux_heap_glibc.c

```diff
@@ -21,6 +21,37 @@
 #define GHT_MAX UT64_MAX
 #endif
 
+static GHT GH(get_va_symbol)(const char *path, const char *symname) {
+	RListIter *iter;
+	RBinSymbol *s;
+	RBin *bin = r_bin_new ();
```

Comment:
what about reusing the instances from core->rbin and core->io instead of creating new ones which make all the plugins load twice?

Reply:
https://github.com/radareorg/radare2/blob/a934a39236b3604e0e6ecebbee2b3b4d9c16bf06/libr/include/r_core.h#L241

I didn't found any rbin inside core, I tried to iterate the binfiles inside bin, but I could only found the path to the program and not the path to the shared library (glibc). Probably I'm missing something.

Reply:
just reuse the core->rbin, and load/close the bin parsing, you dont need to instantiate io because it is already instantiated in core.bin, that will make it faster because loading allt he io and rbin plugins takes a while